.github/workflows/rubocop-filepaths.yml

```diff
@@ -0,0 +1,29 @@
+name: RuboCop filepaths
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  check:
+    runs-on: ubuntu-22.04
+    if: github.repository == 'Homebrew/brew'
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Check public API docs
```

Comment:
Great work! Given this is so quick after checkout (the slowest part): please add to the existing `syntax` job instead, thanks!